docs/settings.rst

```diff
@@ -57,6 +57,80 @@ Example:
 
 .. _here: https://stripe.com/docs/api/python#create_plan
 
+DJSTRIPE_HIERARCHY_PLANS (={})
+===========================
+
+Payment plans levels. 
+
+Allow the user to set levels of access to the plans.
```

Comment:
``Allows you to set levels of access to the plans.``

I'm not a big fan of using ``you``, but ``user`` isn't the right player here - ideally only the developer touches this.